src/ts/contentScripts/create-block/index.tsx

```diff
@@ -0,0 +1,82 @@
+import { browser } from 'webextension-polyfill-ts';
+import {
+  getTopLevelBlockList,
+  ValueElement,
+  getInputEvent,
+  simulateMouseClick,
+  pressESC,
+  pressEnter,
+  pressBackspace,
+  getActiveEditElement,
+  pressShiftTab
+} from '../../utils/dom';
+
+const create = async () => {
+  //get last block from list
+  let lastChild = getTopLevelBlockList().lastChild as ValueElement;
+  let lastBlock = lastChild.querySelector('.roam-block') as ValueElement;
+
+  // click on block to trigger textarea
+  // not needed if lastblock is already selected
+  if (lastBlock) {
+    console.log('lastBlock: ', lastBlock);
+    await simulateMouseClick(lastBlock);
+    // await asyncMouseClick(lastBlock);
+  }
+
+  //mouse click changes dom so we need to get last block again
+  let element = getActiveEditElement() as HTMLTextAreaElement;
+
+  if (element) {
+    console.log('element: ', element);
+    console.log('value', element.value);
+
+    //move to the end of the line
+    element.setSelectionRange(
+      element.value.length * 2,
+      element.value.length * 2
+    );
+
+    // press enter to create new block
+    await pressEnter();
+    // ShiftTab guarantees the new block is a top level block
```

Comment:
only for 1 level of indentation though, no? I imagine that we need to have as many presses of it, as there are indentation levels

Reply:
thst was my intuition too, but follow me
-  `lastBlock` is a top-level block 
   - if it has no children `Enter` creates new top level block
      - `shift tab` is not needed, but does no harm anyway
   - if it has children, enter creates a 2nd level bullet
      - shift-tab makes it the new last top-level bullet

Reply:
conversely, `Tab` after `Enter` guarantees the new block is a child of the previous

Reply:
>if it has children, enter creates a 2nd level bullet 

no mater how many levels of children? that's surprising.